packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const sql = `    
+    SET statement_timeout = 0;
+    SET lock_timeout = 0;
+    SET idle_in_transaction_session_timeout = 0;
+    SET client_encoding = 'UTF8';
+    SET standard_conforming_strings = on;
+    SELECT pg_catalog.set_config('search_path', '', false);
+    SET check_function_bodies = false;
+    SET xmloption = content;
+    SET client_min_messages = warning;
+    SET row_security = off;
+    
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
+    
+    COMMENT ON EXTENSION "uuid-ossp" IS 'generate universally unique identifiers (UUIDs)';
+    
+    CREATE TYPE public.application_provisioningstatus_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    
+    ALTER TYPE public.application_provisioningstatus_enum OWNER TO postgres;
```

Comment:
We probably want to talk about whether this should be `postgres` or `atat_api_admin`.

Reply:
The default is owner is the user who performs the `CREATE TYPE` so we may want to consider whether `ALTER` is required https://www.postgresql.org/docs/13/sql-createtype.html

Reply:
If I had to choose, I would probably go with `atat_api_admin` to be clear and explcit. I am not sure `postgres` adds much value or context.

Reply:
Makes sense to use a more specific name.

Reply:
Changed in c0482b7d84df671b89e575e58c0422fdc5bb6a29